nemo/core/classes/mixins/adapter_mixins.py

```diff
@@ -150,6 +150,20 @@ class AdapterModuleMixin(ABC):
     adapter_global_cfg_key = "global_cfg"
     adapter_metadata_cfg_key = "adapter_meta_cfg"
 
+    def set_accepted_adapters(self, adapters):
```

Comment:
Add type + docstring. Is ti a string, list of string, enum, list of enum ?

Reply:
Also, this should probably be recursive, setting the value for all children of self.